.gitignore

```diff
@@ -2,4 +2,6 @@ node_modules/
 bower_components/
 config/config.js
 rise-financial-version.js
+rise-financial.html
```

Comment:
You can also publish your bower or npm package so that it can contain these built files while leaving them out of the git history.